x-pack/plugins/fleet/common/constants/epm.ts

```diff
@@ -48,6 +48,7 @@ export const GLOBAL_DATA_TAG_EXCLUDED_INPUTS = new Set<string>([
 
 export const PACKAGE_TEMPLATE_SUFFIX = '@package';
 export const USER_SETTINGS_TEMPLATE_SUFFIX = '@custom';
+export const CUSTOM_TEMPLATE_SUFFIX = '@custom';
```

Comment:
nit: could we use the same variable for `USER_SETTINGS_TEMPLATE_SUFFIX`  and `CUSTOM_TEMPLATE_SUFFIX`